packages/apostrophe-widgets/icon-section/src/main.tsx

```diff
@@ -0,0 +1,15 @@
:{"max":"http://localhost:31490/uploads/attachments/clteex752001mqonsbz5h9jom-clsbo2te2021s0i748h3gg7o1-4-full.max.png","full":"http://localhost:31490/uploads/attachments/clteex752001mqonsbz5h9jom-clsbo2te2021s0i748h3gg7o1-4-full.full.png","two-thirds":"http://localhost:31490/uploads/attachments/clteex752001mqonsbz5h9jom-clsbo2te2021s0i748h3gg7o1-4-full.two-thirds.png","one-half":"http://localhost:31490/uploads/attachments/clteex752001mqonsbz5h9jom-clsbo2te2021s0i748h3gg7o1-4-full.one-half.png","one-third":"http://localhost:31490/uploads/attachments/clteex752001mqonsbz5h9jom-clsbo2te2021s0i748h3gg7o1-4-full.one-third.png","one-sixth":"http://localhost:31490/uploads/attachments/clteex752001mqonsbz5h9jom-clsbo2te2021s0i748h3gg7o1-4-full.one-sixth.png","original":"http://localhost:31490/uploads/attachments/clteex752001mqonsbz5h9jom-clsbo2te2021s0i748h3gg7o1-4-full.png"}},"imageAlt":"tekst","title":"3. Gemeente checkt haalbaarheid","description":"Heeft jouw plan op 4 april minimaal 30 likes? Dan checkt de gemeente of jouw plan haalbaar is. Dit doet de gemeente aan de hand van de spelregels. De gemeente kan in deze periode contact met je opnemen over jouw plan.","href":"#","_docId":"clt5vfrwo00050wlj2qgk14wp:en:draft","_edit":true},{"_id":"clteexlvi006o3b6i560tgcp0","metaType":"arrayItem","scopedArrayName":"widget.openstad-iconSection.content","image":{"_id":"clteexnmy001sqonsgfra0i7g","group":"images","createdAt":"2024-03-05T13:35:45.206Z","name":"clsbo2wta02ob0i3w3n3jutr0-6-full","title":"clsbo2wta02ob0i3w3n3jutr0 6 full","extension":"png","type":"attachment","docIds":["clt5vfrwo00050wlj2qgk14wp:en:draft"],"archivedDocIds":[],"length":31670,"md5":"c177a9ee606bcfc1b896b9322b0254c1","width":876,"height":727,"landscape":true,"used":true,"utilized":true,"archived":false,"_isCroppable":true,"_urls":{"max":"http://localhost:31490/uploads/attachments/clteexnmy001sqonsgfra0i7g-clsbo2wta02ob0i3w3n3jutr0-6-full.max.png","full":"http://localhost:31490/uploads/attachments/clteexnmy001sqonsgfra0i7g-clsbo2wta02ob0i3w3n3jutr0-6-full.full.png","two-thirds":"http://localhost:31490/uploads/attachments/clteexnmy001sqonsgfra0i7g-clsbo2wta02ob0i3w3n3jutr0-6-full.two-thirds.png","one-half":"http://localhost:31490/uploads/attachments/clteexnmy001sqonsgfra0i7g-clsbo2wta02ob0i3w3n3jutr0-6-full.one-half.png","one-third":"http://localhost:31490/uploads/attachments/clteexnmy001sqonsgfra0i7g-clsbo2wta02ob0i3w3n3jutr0-6-full.one-third.png","one-sixth":"http://localhost:31490/uploads/attachments/clteexnmy001sqonsgfra0i7g-clsbo2wta02ob0i3w3n3jutr0-6-full.one-sixth.png","original":"http://localhost:31490/uploads/attachments/clteexnmy001sqonsgfra0i7g-clsbo2wta02ob0i3w3n3jutr0-6-full.png"}},"imageAlt":"testtete","title":"4. Verdeel €40.000 over de plannen","description":"Bewoners uit Leidschenveen van 12 jaar en ouder mogen vanaf 13 mei stemmen op de haalbare plannen. Zo bepalen jullie welke plannen uitgevoerd gaan worden.","_docId":"clt5vfrwo00050wlj2qgk14wp:en:draft","_edit":true}]
```

Comment:
Zou je hier alleen de noodzakelijke items in willen zetten (dus niet de apostrophe keys zoals `_id` en `_metaType`) en naar images verwijzen die niet lokaal staan? Dan kan iedereen dit draaien